ci-operator/step-registry/upi/conf/vsphere/ova/windows/upi-conf-vsphere-ova-windows-commands.sh

```diff
@@ -11,7 +11,7 @@ echo "$(date -u --rfc-3339=seconds) - Configuring govc exports."
 source "${SHARED_DIR}/govc.sh"
 
 # TODO: read VM template information from environment variable
-VM_TEMPLATE="windows-golden-images/windows-server-2004-template"
+VM_TEMPLATE="windows-golden-images/windows-server-2022-template-platform-none"
```

Comment:
Before, there was only one vSphere template (windows-golden-images/windows-server-2004-template) for both platform types, vSphere and None. 

Why the need for a specific platform-none one?

Reply:
+1 on question, if this is because docker must be installed, i suggest naming it windows-server-2022-template-platform-none-with-docker

Reply:
+1 on the more specific renaming.

Reply:
Yes, this is because Docker must be installed. How about `windows-server-2022-template-with-docker`. We can delete the `windows-server-2022-template-withDocker` to avoid any confusion. WDYT?

Reply:
Makes sense. Consider including the above explanation in the commit message.

Reply:
What would the explanation be if we are renaming it to `windows-server-2022-template-with-docker`? Isn't it obvious?

>We can delete the windows-server-2022-template-withDocker to avoid any confusion. 

Apparently this cannot be deleted as wee are using it for release-4.10 and below.